osh/hub/osh_xmlrpc/worker.py

```diff
@@ -11,6 +11,11 @@
 from kobo.hub.decorators import validate_worker
 from kobo.hub.models import Task
 
+from kobo.hub.xmlrpc.worker import open_task as kobo_open_task
+
+from osh.hub.errata.models import ScanningSession
```

Comment:
This model was removed in https://github.com/openscanhub/openscanhub/pull/170 at the and of November 2023.  Please, make sure your PR is up-to-date with `main`.